fineract-client/src/main/java/org/apache/fineract/client/util/Calls.java

```diff
@@ -27,7 +27,7 @@
  *
  * @author Michael Vorburger.ch
  */
-public class Calls {
+public final class Calls {
```

Comment:
This makes sense, because it's a utility class with a private constructor... (and I vaguely remember that we have a Checkstyle or ErrorProne or SpotBugs check to enforce this - just want to find out why builds don't show up as red - or where they do)